src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, h, Prop, Watch, State } from "@stencil/core";
+
+@Component({
+  tag: "radio-group",
+  styleUrl: "radio-group.css",
+  shadow: true,
+})
+export class RadioGroup {
+  @Prop({ reflect: true }) list: Array<string>;
+  @State() selectedIndex: number;
+  @State() selectedItem: string;
+
+  handleChange = (e: any) => {
+    this.selectedIndex = parseInt(e.target.value);
+    this.selectedItem = e.target.name;
+  };
+
+  render() {
+    return (
+      <Host>
+        <slot>
+          <form>
+            <div>
+              {this.list.map((item, i) => {
+                return (
+                  <div>
```

Comment:
Do you think there are any HTMLElement types that could be used that have more semantic meaning than a div?

Reply:
`<list>`  make more semantic meaning in this use-case.

Reply:
Cool, yeah lets use that here.